wordpress/wp-content/mu-plugins/cds-base/classes/Modules/Users/Users.php

```diff
@@ -98,7 +98,7 @@ private function getEmailErrors($email)
         }
 
         if (!EmailDomains::isAllowedDomain(sanitize_email($email))) {
-            $error['message'] = __("You can’t use this email domain for registration.");
+            $error['message'] = __("You must enter Government of Canada email to send an invitation.");
             return $error;
```

Comment:
This should probably say "You must enter *a* Government of Canada email...".

Note that this error message is used for both of these cases:
- badly-formatted email addresses: anything that would fail a generic email validation (eg, `hello`)
- properly-formatted email addresses with a non-GC domain (`hello@canada.com`)

Reply:
👍